packages/block-library/src/accordion-item/edit.js

```diff
@@ -102,11 +102,6 @@ export default function Edit( {
 					label={ __( 'Settings' ) }
 					resetAll={ () => {
 						setAttributes( { openByDefault: false } );
-						if ( contentBlockClientId ) {
```

Comment:
We should also remove the attribute passed in `template` in `useInnerBlocksProps` above.

Reply:
I think so too, since that context never has value. Was double-checking in the original PR - https://github.com/WordPress/gutenberg/pull/71895#discussion_r2644946896.

Reply:
I was mostly referring to `openByDefault` for panel, but it seems we might not need the other one too.